app/styles/_story.scss

```diff
@@ -0,0 +1,21 @@
+article section {
+  a {
+    border-bottom: .08em solid rgba(black, .1);
+  }
+  
+  h2 {
+    margin-bottom: 1rem;
+  }
+  
+  h3, h4 {
+    margin-bottom: 0;
+  }
+  
+  blockquote {
+    border-left: 4px solid $nearwhite;
+    padding-left: 20px;
+    font-size: 18px;
+    line-height: 1.6em;
+    font-weight: bold;
+  }
+}
```

Comment:
let's put this in the legacy folder since the story page is due for a revamp to strictly ember